compiler/ir/backend.js/src/org/jetbrains/kotlin/ir/backend/js/transformers/irToJs/JsIntrinsicTransformers.kt

```diff
@@ -319,5 +322,27 @@ private inline fun MutableMap<IrSymbol, IrCallTransformer>.withTranslatedArgs(
     function: IrSimpleFunctionSymbol,
     crossinline t: (List<JsExpression>) -> JsExpression
 ) {
-    put(function) { call, context -> t(translateCallArguments(call, context)) }
+    put(function) { call, context ->
+        runCatching {
```

Comment:
Please prefer to use standard control flow structures instead of library functions, especially if they don't add much value as in this case and below. `try { ... } catch(...) { ... }` looks fine and is far friendlier to debuggers than `runCatching` + `getOrElse`.

Reply:
Rewrote